src/store/app.ts

```diff
@@ -114,10 +114,9 @@ export const leaderAtom = atom(
     })),
 );
 
-export const networkConfigAtom = atomWithStorage(
-  'agoric-network-config',
-  networkConfigs.mainnet,
-);
+export const networkConfigAtom = import.meta.env.VITE_NETWORK_CONFIG_URL
```

Comment:
Needs documentation.

Why does it have VITE?

The name should convey that it's optional. Eg "FORCE_" to indicate that it overrides the UI config

Reply:
I guess this PR also doubles as a design proposal, so apologies that it seems out of the blue.

Updated the readme, does that make it clear enough?

Re: VITE prefix, see  https://vitejs.dev/guide/env-and-mode.html#env-files -- "To prevent accidentally leaking env variables to the client, only variables prefixed with VITE_ are exposed to your Vite-processed code"